index.html

```diff
@@ -1,15 +1,141 @@
 <!DOCTYPE html>
-<html >
+<html lang="en-US">
 	<head>
 		<meta charset="UTF-8">
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
-		<title></title>
-		<style >
-			
-		</style>
+		<title>Intuit. Signup</title>
 		<link rel="stylesheet" href="stylesheet.css" />
 	</head>
 	<body>
-		<h1>Intuit Login Page clone</h1>
+		<section>
+			<nav id="navbar">
+				<div class="link_container">
+					<div class="left_nav">
+						<div>Already have an Intuit account?</div>
+						<a href="#" class="link">I forgot my User ID or Password</a>
+					</div>
+					<div class="right_nav">
+						<button id="signin_button">
+						<span class="lock"></span>
+						Sign Up
+						</button>
+					</div>
+				</div>
+			</nav>
+			<header class="header">
+				<div class="title">
+					<a href="#"></a>
+				</div>
+			</header>
+			<section>
+				<div>
+					<header class="head">
+						<div>
+							<ul>
+								<li class="list_item list_icon" id="mint_icon"></li>
+								<li class="list_item list_text" id="mint_text"></li>
+								<li class="list_item list_icon" id="qb_icon"></li>
+								<li class="list_item list_text" id="qb_text"></li>
+								<li class="list_item list_icon" id="cb_icon"></li>
+								<li class="list_item list_text" id="cb_text"></li>
+							</ul>
+						</div>
+					</header>
+				</div>
+				<div class="main">
+					<section >
+						<header class="form_header">
+							<span class="form_title">Create an Intuit account</span>
+							<span class="form_title_des">
+								<span class="div">One account for everything Intuit, including Mint.
+									<a href="#" class="link">Learn more</a>
+								</span>
+							</span>
+						</header>
+						<form>
+							<fieldset>
+								<label class="label">Email address</label>
+								<div class="form_div">
+									<input type="email" name="email" class="form_input" />
+								</div>
+								<span class="input_des"></span>
+							</fieldset>
+							<fieldset>
+								<label class="label">Phone (recommended)</label>
+								<div class="form_div">
+									<input type="text" name="number" class="form_input" />
+								</div>
+								<span class="input_des">Standard call, messaging or data rates may apply.</span>
+							</fieldset>
+							<fieldset>
+								<div class="form_div">
+									<label class="label">Password</label>
+								</div>
+								<div class="form_div">
+									<input type="password" name="password" class="form_input" />
+									<span class="password_lock"></span>
+								</div>
+								<span class="input_des"></span>
+							</fieldset>
+							<div class="submit_div">
+								<button class="submit_button">
```

Comment:
Note that the `type` attribute of a button is important, you can associate the button with the form by giving it a `type` attribute of `submit` or make it as it is by giving it an attribute of `button`.  Learn more here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button

Reply:
Thanks for your comment. I'll check the link and add the appropriate type to the input.